includes/frontend.php

```diff
@@ -447,6 +456,16 @@ public function register_scripts() {
 			true
 		);
 
+		wp_register_script(
```

Comment:
Actually, it's better to register it only when enabled. You can skip the 'register' level and immediately do 'enqueue' with all the parameters.